tests/pages/services/ServiceTable-cy.js

```diff
@@ -268,6 +268,14 @@ describe("Service Table", function() {
         .contains("stop")
         .should("not.exist");
     });
+
+    it("DCOS_SERVICE_WEB_PATH adds Open Service item", function() {
```

Comment:
i did not grok that on first reading. something like `has an 'Open Service'-DropdownItem when DCOS_SERVICE_WEB_PATH-label is set` would have helped me.